hw/xbox/dsp/dsp_dma.c

```diff
@@ -241,13 +242,35 @@ static void dsp_dma_run(DSPDMAState *s)
                 }
             }
 
-            // write to scratch memory
-            s->scratch_rw(s->scratch_rw_opaque,
-                scratch_buf, scratch_addr, count*item_size, 1);
+            /* FIXME: Move to function; then reuse for both directions */
+            switch (buf_id) {
+            case 0x0:
```

Comment:
Prefer named constants here

Reply:
Agreed, but can be done later. This is only a small refactor of the existing code.

I also tried naming them in my unit tests and ended up with renaming these constants over and over as I found out new stuff (and some things are still not known).